train_gpt2.c

```diff
@@ -10,14 +10,22 @@ There will be other versions of this code that specialize it and make it fast.
 
 #include <stdio.h>
 #include <stdlib.h>
+#ifdef _WIN32
+#define _USE_MATH_DEFINES
+#endif
 #include <math.h>
 #include <time.h>
 #include <string.h>
-#include <unistd.h>
-#ifdef OMP
+#if defined(OMP) && !(_OPENMP < 202011)
```

Comment:
Note that I don't think we need to manually define OMP anymore with this version check that is automatically defined if you depend on OpenMP. Not sure you even need it in the manual Makefile